### ISSUE TITLE
Director Health Check is still pending

### ISSUE THREAD

**Issue by phil-or** [Icinga/icingaweb2-module-director#1876]
## Expected Behavior

In our Setup, we configured the director-health-check as a Service.
After the director Jobs have been successfully completed and the config was deployed, the director-health Service should also be displayed as "OK"

![image](https://user-images.githubusercontent.com/36534023/58153512-7fde4d00-7c6f-11e9-9031-772a1e8e5646.png)

## Current Behavior

After the director Jobs have been successfully completed and the config was deployed, the director-health Service changed to "WARNING". 
But there is no real problem recognizable...

![image](https://user-images.githubusercontent.com/36534023/58153814-52de6a00-7c70-11e9-9e65-493d98cf0761.png)

If you click to "Icinga Director" - "Deployements" and then check the service again, the state change to "OK"

## Steps to Reproduce (for bugs)

Create a director-health Service and deploy configs over configured Director Jobs

## Your Environment

* Director version (System - About): master
* Icinga Web 2 version and modules (System - About): 2.6.3
* Icinga 2 version (`icinga2 --version`): r2.10.4-1
* Operating System and version: CentOS7
* Webserver, PHP versions: Apache

**Comment by smalik03** [comment]
I am having the same issue. Any ideas as to why this would be happening?

**Comment by ix-dev** [comment]
We are facing the same issue and noticed some strange update behavior in the `director_deployment_log` table.

Steps to manually reproduce the issue:
1. Click on "Deploy pending changes" in icingaweb2
2. Close the tab immediately to avoid redirection to the "Deployments" overview

Now, in the `director_deployment_log` table the columns `stage_collected`, `startup_succeeded` and `startup_log` are set to `NULL`. This does not change until you open the "Deployments" overview in icingaweb2. When you open it the values are immediately updated and the Health check reports "OK" again.

We think that the Health Check relies on this information to check if the deployments is working.

We observed this behavior when we trigger the deployment manually as described above and also when the deployment is done by a config job. We did not test it using the REST API. Furthermore, we are not sure how it behaves in case the deployment fails.
